docs/releases/release-1.0.md

```diff
@@ -0,0 +1,50 @@
+# Kubeflow Pipelines 1.0 Release Notes
+
+## Versioning policy
+
+Kubeflow Pipelines 1.0 is the first major release of Kubeflow Pipelines. It follows the Kubeflow Pipelines [versioning policy](https://github.com/kubeflow/pipelines/blob/master/docs/releases/versioning-policy.md).
+
+## Deployment options
+
+There are two available ways to [deployment Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+
+* [Standalone Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/), which allows users the maximum flexibility in configuring the Kubernetes clusters and a Kubeflow Pipelines instance.
+
+* [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs), which minimizes the user effort in deploying a Kubeflow pipelines instance.
+
+## Core features and APIs
+
+Core features and APIs are the fundamental functionalities of Kubeflow Pipelines. The most fundamental functionality is to manage the Kubeflow Pipelines resources (that is, pipeline, pipeline version, run, job, experiment) via the operations listed below. Active supports are available for core features and APIs.
```

Comment:
I think we need to add component spec here:

> ComponentSpec
We definitely want to promise backwards compatibility for ComponentSpec. https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/components/structures/components.json_schema.json

reference: https://github.com/kubeflow/pipelines/issues/3311#issuecomment-601009544